errors/error-handler.js

```diff
@@ -0,0 +1,17 @@
+exports.psqlErrorHandler = (err, req, res, next) => {
+  if (err?.code) {
+    res.status(400).send({ msg: "Bad Request" });
+  }
+  next(err);
```

Comment:
If the error has a code you are sending a response as well as calling next. Both of these things shouldn't happen.